hack/run-gendocs.sh

```diff
@@ -45,8 +45,7 @@ kube::util::gen-doc "${genman}" "${KUBE_ROOT}" "docs/man/man1"
 kube::util::gen-doc "${genbashcomp}" "${KUBE_ROOT}" "contrib/completions/bash/"
 kube::util::gen-analytics "${KUBE_ROOT}"
 
-"${mungedocs}" "--root-dir=${KUBE_ROOT}/docs/"
-ret=$?
+"${mungedocs}" "--root-dir=${KUBE_ROOT}/docs/" && ret=0 || ret=$?
```

Comment:
It is evaluated from left to right. This is required because of set -e. As it was, the script would exit immediantly when mungedocs exits non 0. I believe this is the best way of capturing error codes in errexit scripts